hedera-node/src/main/java/com/hedera/services/legacy/services/state/AwareProcessLogic.java

```diff
@@ -226,12 +225,12 @@ private SignatureStatus rationalizeWithPreConsensusSigs(TxnAccessor accessor) {
 		return sigStatus;
 	}
 
-	void addForStreaming(
-			com.hederahashgraph.api.proto.java.Transaction grpcTransaction,
-			ExpirableTxnRecord expirableTxnRecord,
-			Instant consensusTimeStamp
+	void stream(
+			com.hederahashgraph.api.proto.java.Transaction txn,
+			ExpirableTxnRecord record,
```

Comment:
This might give a Code smell in SonarCloud as `record` is a restricted identifier

Reply:
Yep! Pushed fixes for all code smells (other than deleting copyright headers 😁 )